applications/DEMApplication/custom_constitutive/DEM_global_damping_nonviscous_constantforcedir.cpp

```diff
@@ -0,0 +1,53 @@
+//  Kratos Multi-Physics - DEM Application
+//
+//  License:       BSD License
+//                 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Rafael Rangel (rrangel@cimne.upc.edu)
+//
+
+// Project includes
+#include "DEM_global_damping_nonviscous_constantforcedir.h"
+#include "custom_utilities/GeometryFunctions.h"
+
+namespace Kratos
+{
+    DEMGlobalDampingModel::Pointer DEMGlobalDampingNonViscousCteForceDir::Clone() const {
```

Comment:
NICE! Those new names are much clearer now. Only one point, for "DEMGlobalDampingNonViscousCteForceDir", I am confused about "Cte". For "Constant", a commonly used short name is "Const". Or maybe "Cst" if you prefer to use three letters, although it's less common. But "Cte" may be a bit hard to understand.

Reply:
True, 'cte' is appropriate in Spanish

Reply:
Nice done!